logprep/processor/requester/processor.py

```diff
@@ -0,0 +1,112 @@
+"""
+Requester
+=========
+
+A processor to invoke http requests. Could be usefull to enrich events from an external api or
+to trigger external systems by and with event field values.
+
+Example
+^^^^^^^
+..  code-block:: yaml
+    :linenos:
+
+    - requestername:
+        type: requester
+        specific_rules:
+            - tests/testdata/rules/specific/
+        generic_rules:
+            - tests/testdata/rules/generic/
+
+"""
+import json
+import re
+import requests
+
+from logprep.abc import Processor
+from logprep.processor.base.exceptions import DuplicationError
+from logprep.processor.requester.rule import RequesterRule
+from logprep.util.helper import add_field_to, get_dotted_field_value, get_source_fields_dict
+
+TEMPLATE_KWARGS = ("url", "json", "data", "params")
+
+
+class Requester(Processor):
+    """A processor to invoke http requests with field data
+    and parses response data to field values"""
+
+    rule_class = RequesterRule
+
+    def _apply_rules(self, event, rule):
+        source_field_dict = get_source_fields_dict(event, rule)
+        self._check_for_missing_values(event, rule, source_field_dict)
+        kwargs = self._template_kwargs(rule.kwargs, source_field_dict)
+        rsp = self._request(event, rule, kwargs)
+        self._handle_response(event, rule, rsp)
+
+    def _handle_response(self, event, rule, rsp):
+        if rule.target_field:
+            result = self._get_result(rsp)
```

Comment:
both if cases make use of this line, maybe we can do it once in the first line of this method?